api/odigos/v1alpha1/instrumentedapplication_types.go

```diff
@@ -51,6 +59,13 @@ type RuntimeDetailsByContainer struct {
 	EnvVars        []EnvVar                   `json:"envVars,omitempty"`
 	OtherAgent     *OtherAgent                `json:"otherAgent,omitempty"`
 	LibCType       *common.LibCType           `json:"libCType,omitempty"`
+
+	// Stores the error message from the CRI runtime if returned to prevent instrumenting the container if an error exists.
+	CriErrorMessage *string `json:"criErrorMessage,omitempty"`
+	// Holds the environment variables retrieved from the container runtime.
+	EnvFromContainerRuntime []EnvVar `json:"envFromContainerRuntime,omitempty"`
+	// A temporary variable used during migration to track whether the new runtime detection process has been executed. If empty, it indicates the process has not yet been run. This field may be removed later.
+	RuntimeUpdateState *ProcessingState `json:"runtimeUpdateState,omitempty"`
```

Comment:
these are from main, so I guess the `source-crd` branch needs to be updated to main

Reply:
Yeah, I merged `main` into `source-crd`